dashboard/src/scenes/person/components/PersonDocuments.js

```diff
@@ -99,7 +99,7 @@ const PersonDocuments = ({ person }) => {
                 setDocumentToEdit(doc);
               }}>
               <td className="tw-p-3">
-                <p className="tw-m-0 tw-flex tw-items-center tw-overflow-hidden tw-break-all tw-font-bold">
+                <p className="tw-m-0 tw-flex tw-items-center tw-overflow-hidden tw--all tw-font-bold">
```

Comment:
Le `tw--all` n'a pas de sens si ?

```suggestion
                <p className="tw-m-0 tw-flex tw-items-center tw-overflow-hidden tw-font-bold">
```

Reply:
Tout à fait ça n'a aucun sens !! C'est une erreur. Je change !!